parser/sql/dialect/oracle/src/main/antlr4/imports/oracle/DDLStatement.g4

```diff
@@ -427,7 +427,7 @@ alterTableProperties
     | parallelClause
     | rowMovementClause
     | logicalReplicationClause
-    | flashbackArchiveClause)+ | renameTableSpecification) alterIotClauses? alterXMLSchemaClause?
+    | flashbackArchiveClause)+ | renameTableSpecification)? alterIotClauses? alterXMLSchemaClause?
```

Comment:
SQL> ALTER TABLE mytable ADD OVERFLOW TABLESPACE admin_tbs2;
ALTER TABLE mytable ADD OVERFLOW TABLESPACE admin_tbs2
            *
ERROR at line 1:
ORA-28658: This operation is supported only for Index-Organized tables